### ISSUE TITLE
Remove simple/advanced search toggle

### ISSUE THREAD

**Issue by Arantor** [SimpleMachines/SMF#1331]
Right now there is a 'simple' and 'advanced' search setup. The idea is that you can have a 'simple' search that just presents you with the search box and that you would press a button to go to the 'advanced' box. Or you could just disable that and always go to the advanced search.

I see no reason to have the option at all and would prefer we just dispensed with it and had 'advanced search' permanently enabled, this would affect both main and PM searching the same way.

**Comment by ChainyEO** [comment]
I think that might well be a good idea. Simple Search would still remain in the top right-hand corner, I suppose. Then the Search link in the Main Menu would be reserved for the Advanced Search. I'm now wondering why anyone would want the Simple Search function in both of these locations at the same time? (maybe some themes might disable the top right-hand corner Simple Search and so then people might like the ability to switch Simple Search on for the Main Menu item?)

The current setting for switching Simple Search on/off in the Main Menu is a bit confusing because it just says "Enable Simple Search". At first I wondered what it was referring to because Simple Search seemed to be enabled by default (top right corner). It was only after playing with it a little that I realised that it was referring only to the Search function within the Main Menu... For this reason, I actually added a slightly more detailed explanation to the Esperanto translation (English version of my translation):

> If this option is enabled, clicking on Search in the main menu will lead users to a simple search field with a link to more advanced search options.

This contrasts with the original English:

>  This will show a simple search form and a link to a more advanced form.

In the latter it's not really clear what is meant.
